test/e2e/README.md

```diff
@@ -60,6 +64,28 @@ T_TINKERBELL_INVENTORY_CSV # path to hardware-inventory.csv file
 T_TINKERBELL_SSH_AUTHORIZED_KEY # ssh public key for connectioning to machines
 ```
 
+# CloudStack tests requisites
+
+The following env variables need to be set:
+```
+T_CLOUDSTACK_DOMAIN
+T_CLOUDSTACK_ZONE
+T_CLOUDSTACK_ACCOUNT
+T_CLOUDSTACK_NETWORK
+T_CLOUDSTACK_MANAGEMENT_SERVER
+T_CLOUDSTACK_SSH_AUTHORIZED_KEY
+T_CLOUDSTACK_TEMPLATE_REDHAT_1_20
+T_CLOUDSTACK_TEMPLATE_REDHAT_1_21
+T_CLOUDSTACK_COMPUTE_OFFERING_LARGE
+T_CLOUDSTACK_COMPUTE_OFFERING_LARGER
```

Comment:
Why do we have these two env vars?

Reply:
That would be for supporting upgrade from the smaller compute offering to the larger one as a test case

Reply:
Can we just make these a part of the test value then instead of as an env var? It doesn't seem necessary to have these as env vars to me, but maybe we wanted to abstract it?